ttkthemes/_utils.py

```diff
@@ -21,6 +22,40 @@ def is_python_3():
     return sys.version_info.major == 3
 
 
+@contextlib.contextmanager
+def temporary_chdir(new_dir):
+    """Like os.chdir(), but always restores the old working directory.
```

Comment:
While PEP-8 gives the first line of the docstring on the same line as the quotes as the multi-line docstring example, it actually also allows putting the actual line on a new line. I would prefer that, as it would be in line with the style of the rest of the repository.

Reply:
As in the second line after that one of PEP-257:
>The summary line may be on the same line as the opening quotes or on the next line.

It is ambiguous on whether this applies to only lines that are otherwise three characters too long, or to all first lines of a multi-line docstring. Though, admittedly, I usually don't use summaries and extensive descriptions but parameter lists as generated by PyCharm.